cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfLinkComponent.cpp

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfLinkComponent.h"
+
+#include "CoreUtils/Assert.h"
+#include "CoreUtils/Config.h"
+#include "UrdfParser.h"
+
+void UUrdfLinkComponent::initialize(UrdfLinkDesc* link_desc)
+{
+    // for now link with no visual node or multiple visual node is not supported
+    ASSERT(link_desc->visual_descs_.size() == 1);
+
+    const UrdfVisualDesc& visual_desc = link_desc->visual_descs_[0];
+    const UrdfGeometryDesc& geometry_desc = visual_desc.geometry_desc_;
+
+    UStaticMesh* static_mesh = nullptr;
+    FVector type_scale;
```

Comment:
Rename to `relative_scale` or `relative_scale_3d` to match what Unreal calls it.

Reply:
fixed